pom.xml

```diff
@@ -45,7 +50,7 @@
         <dependency>
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.user.core</artifactId>
-            <version>4.4.3</version>
+            <version>4.5.1</version>
```

Comment:
Bumping this dependency will make this extension not backward compatible. 
is this expected?